decidim-core/app/assets/stylesheets/decidim/modules/_collapsible-list.scss

```
@@ -36,7 +36,7 @@
 
   .collapsible-list__see-more{
     display: none;
-    color: #599aa6;
+    color: var(--secondary);
```

```suggestion
    color: $secondary;
```
Although I think the personalized secondary color should be applied here, it also should be done in the rest of the anchors. This is not currently the case, so, for the sake of consistence, we might want to use the SASS variable instead.

---

following the previous comment @tramuntanal , I think this is the suggestion that should be applied for now (until there's another PR that improves admin customization for secondary color links).